scripts/build_deploy.sh

```diff
@@ -1,13 +1,11 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -e
 
 # see Makefile for the IMG_ variables semantic
 IMG_REGISTRY=""
 IMG_ORG="${IMG_ORG:-kubeflow}"
 IMG_REPO="${IMG_REPO:-model-registry}"
-DOCKER_USER="${DOCKER_USER}"
-DOCKER_PWD="${DOCKER_PWD}"
```

Comment:
isn't this conventional to make it more evident these are the env variables required when someone is invoking this script, for instance from make/gha ?

Reply:
I wasn't very sure tbh, we can keep them.